bin/fixmystreet.com/add_emergency_message

```diff
@@ -35,27 +40,44 @@ if (!$opts->commit) {
     say colored("*** DRY RUN ***", 'cyan');
 }
 
-my $field = {
-    order => 0,
-    required => 'true',
-    protected => 'true',
-    code => $opts->code,
-    description => $opts->question,
-    datatype => 'singlevaluelist',
-    variable => 'true',
-    values => [
-        {
-            key => 'yes',
-            name => $opts->yes,
-            disable => 1,
-            disable_message => $opts->message,
-        },
-        {
-            key => 'no',
-            name => $opts->no,
-        }
-    ],
-};
+my $field;
+
+if ( $opts->mode eq 'questions' ) {
+    say STDERR "questions, code, yes and no required"
+        unless $opts->questions && $opts->code && $opts->yes && $opts->no;
+
+    $field = {
+        order => 0,
+        required => 'true',
+        protected => 'true',
+        code => $opts->code,
+        description => $opts->question,
+        datatype => 'singlevaluelist',
+        variable => 'true',
+        values => [
+            {
+                key => 'yes',
+                name => $opts->yes,
+                disable => 1,
+                disable_message => $opts->message,
+            },
+            {
+                key => 'no',
+                name => $opts->no,
+            }
+        ],
+    };
+} else {
+    $field = {
+        order => 0,
+        required => 'true',
+        protected => 'true',
+        code => '_fms_disable_',
+        description => $opts->message,
+        datatype => 'notice',
```

Comment:
datatype is either string or singlevaluelist (for question, ignored for others now, so I think can be left off).